resource/tasks.json

```diff
@@ -3136,9 +3109,22 @@
             "WaitAfterPRTS"
         ]
     },
+    "StartButton3": {
+        "baseTask": "StartButton2",
+        "next": [
+            "StartButton3TryAgain",
+            "StartButton3WaitTime"
+        ]
+    },
+    "StartButton3TryAgain": {
+        "baseTask": "StartButton2TryAgain",
+        "template": "StartButton3.png",
+        "next": [
+            "StartButton3#next"
+        ]
+    },
     "StartButton3WaitTime": {
-        "algorithm": "JustReturn",
-        "postDelay": 40000,
+        "baseTask": "StartButton2WaitTime",
         "next": [
             "PRTS#next",
             "EndOfAction",
```

Comment:
Tyvm for the heads up, one more question. how does the baseTask interpret the "next" inheritance. For example in this case can I just use `"baseTask": "StartButton2WaitTime",` and `next: ["StartButton3TryAgain"]` because it inherits the other options such as PRTS#Next, EndOfAction etc... Or I need to repeat all of the next like here.

Reply:
The "next" field needs to be written in its entirety. Whenever a display description is provided, it will directly override the previous configuration.

Reply:
Oh completely override, perfect than!
Thanks.